apis/dataplex/v1alpha1/entrygroup_types.go

```diff
@@ -15,26 +15,21 @@
 package v1alpha1
 
 import (
+	"github.com/GoogleCloudPlatform/k8s-config-connector/apis/common/parent"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/apis/k8s/v1alpha1"
+
 	// GCP Resource Reference type.
-	refv1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var DataplexEntryGroupGVK = GroupVersion.WithKind("DataplexEntryGroup")
 
-// The Parent resource that the DataplexEntryGroup resource resides in.
-type DataplexEntryGroupParent struct {
-	// +required
-	ProjectRef *refv1beta1.ProjectRef `json:"projectRef"`
-	// +required
-	Location string `json:"location"`
-}
-
 // DataplexEntryGroupSpec defines the desired state of DataplexEntryGroup
 // +kcc:spec:proto=google.cloud.dataplex.v1.EntryGroup
 type DataplexEntryGroupSpec struct {
-	DataplexEntryGroupParent `json:",inline"`
+	ParentRef *parent.ProjectAndLocationRef `json:",inline"`
```

Comment:
I would still argue that it's simpler just to put the fields in here directly, but ... not a blocker